dojo/db_migrations/0047_sla_notifications.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.14 on 2020-07-10 09:34
+
+from django.db import migrations, models
+import multiselectfield.db.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0046_endpoint_status'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='jira_conf',
+            name='sla_notification',
+            field=models.BooleanField(default=True, verbose_name='Send SLA notifications as comment?'),
```

Comment:
I was thinking this could somehow be "overridden" through the notification checkbox on the product view page. If unchecked, then the notifications would not be sent. Could be checked by default, something like that.

Reply:
So actually, if nobody set notifications anywhere, no one would get them, which is what was intended I guess.
However, it will still go through all the findings -- potentially for nothing.

Reply:
But probably makes more sense.. will try something for this then:

![image](https://user-images.githubusercontent.com/5468769/87329564-70872680-c537-11ea-9097-6883b389a76c.png)

Reply:
I think that's the better level to have it at.

Reply:
Done. Product level notification will override the global JIRA setting.